sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/implementation/models/EdgeNGramTokenFilter.java

```diff
@@ -16,7 +16,7 @@
  * of an input token. This token filter is implemented using Apache Lucene.
  */
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "@odata.type")
-@JsonTypeName("#Microsoft.Azure.Search.EdgeNGramTokenFilter")
+@JsonTypeName("#Microsoft.Azure.Search.EdgeNGramTokenFilterV2")
```

Comment:
This might cause serialization issues to the service as we changes the type name here.

Reply:
Right, this change is not needed. Revert

Reply:
Will revert all implementation model changes.